spec/models/reports/performance_score_spec.rb

```diff
@@ -102,6 +102,12 @@
       @opd_load = 0
       expect(perf_score.registrations_rate).to eq(0)
     end
+
+    it "maxes at 100 if registrations exceeds target" do
+      @registrations = 500
+      @opd_load = 100
```

Comment:
Ivars for test setup? #derty 😅 

TBH it's quite neat.

Reply:
It's because otherwise I'd have to construct a new `double` for every test. If you know a better way please show! :)